libs/payments/ui/src/lib/utils/en.ftl

```diff
@@ -0,0 +1,43 @@
+# Checkout start
+metadata-title-checkout-start = Checkout | { $productTitle }
+metadata-description-checkout-start = Enter your payment details to complete your purchase.
+
+# Checkout processing
+metadata-title-checkout-processing = Processing | { $productTitle }
+metadata-description-checkout-processing = Please wait while we finish processing your payment.
+
+# Checkout error
+metadata-title-checkout-error = Error | { $productTitle }
+metadata-description-checkout-error = There was an error processing your subscription. If this problem persists, please contact support.
+
+# Checkout success
+metadata-title-checkout-success = Success | { $productTitle }
+metadata-description-checkout-success = Congratulations! You have successfully completed your purchase.
+
+# Checkout needs_input
+metadata-title-checkout-needs-input = Action required | { $productTitle }
+metadata-description-checkout-needs-input = Please complete the required action to proceed with your payment.
+
+# Upgrade start
+metadata-title-upgrade-start = Upgrade | { $productTitle }
+metadata-description-upgrade-start = Enter your payment details to complete your upgrade.
+
+# Upgrade processing
+metadata-title-upgrade-processing = Processing | { $productTitle }
+metadata-description-upgrade-processing = Please wait while we finish processing your payment.
+
+# Upgrade error
+metadata-title-upgrade-error = Error | { $productTitle }
+metadata-description-upgrade-error = There was an error processing your upgrade. If this problem persists, please contact support.
+
+# Upgrade success
+metadata-title-upgrade-success = Success | { $productTitle }
+metadata-description-upgrade-success = Congratulations! You have successfully completed your upgrade.
+
+# Upgrade needs_input
+metadata-title-upgrade-needs-input = Action required | { $productTitle }
+metadata-description-upgrade-needs-input = Please complete the required action to proceed with your payment.
+
+# Default
+metadata-title-default = Default | { $productTitle }
+metadata-description-default = Mozilla!!
```

Comment:
Is this intended to be user facing? If not we probably don't need to localize this.
If so, should we make the description more meaningful? (At the very least we would want to use the brand-term.)